http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -167,6 +166,7 @@ private SdkEventLoopGroup nonManagedEventLoopGroup(SdkEventLoopGroup eventLoopGr
     @Override
     public void close() {
         sdkEventLoopGroup.eventLoopGroup().shutdownGracefully();
```

Comment:
Should we do a runOrLogError in case the shutdown fails so we make sure we cleanup the pool?

Reply:
+1